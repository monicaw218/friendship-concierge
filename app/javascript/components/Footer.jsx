import React from 'react';
import { Layout, Menu } from 'antd';
import { Link } from 'react-router-dom';
const { Footer } = Layout;

export default () => (
  <Footer style={{ textAlign: 'center' }}>Weitekamp Â©2022.</Footer>
);
