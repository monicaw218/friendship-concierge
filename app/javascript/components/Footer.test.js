import renderer from 'react-test-renderer';
import Footer from './Footer.jsx';

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
