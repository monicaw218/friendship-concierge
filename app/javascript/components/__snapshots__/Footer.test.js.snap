// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  Weitekamp Â©2022.
</footer>
`;
